java-client/src/main/java/co/elastic/clients/elasticsearch/_helpers/bulk/BulkIngester.java

```diff
@@ -483,13 +492,25 @@ public Builder<Context> flushInterval(long value, TimeUnit unit) {
         /**
          * Sets an interval flushing any bulk actions pending if the interval passes. Defaults to not set.
          * <p>
-         * Flushing is still subject to the maximum number of requests set with {@link #maxConcurrentRequests}.     
+         * Flushing is still subject to the maximum number of requests set with
+         * {@link #maxConcurrentRequests}.
+         * Deprecated in favor of {@link #scheduler}
```

Comment:
Nit: use the Javadoc `@deprecated` tag, and link to the method instead of the private field:

```java
@deprecated use {@link #scheduler(ScheduledExecutorService)}
```